src/borg/archiver/create_cmd.py

```diff
@@ -661,6 +662,16 @@ def build_parser_create(self, subparsers, common_parser, mid_common_parser):
         """
         )
 
+        class MakePathSafeAction(argparse.Action):
+            def __call__(self, parser, namespace, path, option_string=None):
+                try:
+                    sanitized_path = make_path_safe(path)
+                except ValueError as e:
+                    raise argparse.ArgumentError(self, e)
+                if sanitized_path == ".":
+                    raise argparse.ArgumentError(self, f"{path!r} is not a valid file name")
+                setattr(namespace, self.dest, sanitized_path)
+
```

Comment:
quite some argument parsing helpers are already in `helpers.parseformat`.

Reply:
Moved